can/util.py

```diff
@@ -190,10 +190,39 @@ def load_config(path=None, config=None):
     if 'bitrate' in config:
         config['bitrate'] = int(config['bitrate'])
 
-    can.log.debug("loaded can config: {}".format(config))
+    can.log.debug("can config: {}".format(config))
     return config
 
 
+def choose_socketcan_implementation():
```

Comment:
This shouldn't be in a doc update PR

Reply:
This branch that you created quite a while back was a few dozen commits behind `develop`, and this resulted from the rebase to resolve the conflicts. Was this wrong?

Reply:
Oh right I didn't notice you were starting with an older branch. Yes this was removed on purpose (in the socketCAN consolidation).

I suggest rebasing on top of develop - either force push over this branch or to another branch.

Reply:
If it helps I've rebased my original clean up commits on the current develop branch `fix-sphinx-errors-rebase`

Reply:
Uhm, I already applied other patches on top of the rebase. Wouldn't it be the simplest to just remove the `choose_socketcan_implementation()` function?

Reply:
Sure

Reply:
Done.